pytorch_lightning/utilities/cli.py

```diff
@@ -104,6 +105,11 @@ def __init__(self, optimizer: Optimizer, monitor: str, *args: Any, **kwargs: Any
         self.monitor = monitor
 
 
+LRSchedulerTypeTuple = (torch.optim.lr_scheduler._LRScheduler, ReduceLROnPlateau)
```

Comment:
If the CLI requires the ReduceLROnPlateau defined in this module, then it shouldn't accept the one from pytorch. Though this duplicates these three variables defined in pytorch_lightning.utilities.types.

Reply:
Can you add a comment?

Reply:
Done